packages/client/lib/config.ts

```diff
@@ -184,15 +184,25 @@ export class Config {
   }
 
   /**
-   * Returns the directory for storing the client sync data
+   * Returns the directory for storing the client chain data
    * based on syncmode and selected chain (subdirectory of 'datadir')
    */
-  getSyncDataDirectory(): string {
-    const syncDirName = this.syncmode === 'light' ? 'lightchaindata' : 'chaindata'
-    const chain = this.common.chainName()
-    const networkDirName = chain === 'mainnet' ? '' : `${chain}/`
+  getChainDataDirectory(): string {
+    const networkDirName = this.common.chainName()
```

Comment:
How would this work for private chains?

Reply:
Hmm, I haven't spent too much time thinking about custom chains yet TBH. On a first look it is possible though to provide a chain name in `Common.forCustomChain()` and this would then be used for the folder name.